platforms/core-configuration/model-core/src/main/java/org/gradle/api/internal/provider/Collectors.java

```diff
@@ -183,9 +182,15 @@ private static <T> void visitValue(Action<? super ValueSupplier.ExecutionTimeVal
 
     public static class ElementsFromCollection<T> implements Collector<T> {
         private final Iterable<? extends T> value;
+        private final SideEffect<? super Iterable<? extends T>> sideEffect;
```

Comment:
💅 add `@Nullable` if this field survives the next iteration?

Reply:
Removed this change